Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -506,6 +506,9 @@ extension TelemetryWrapper {
             GleanMetrics.DefaultBrowserOnboarding.dismissPressed.add()
         case (.action, .tap, .goToSettingsDefaultBrowserOnboarding, _, _):
             GleanMetrics.DefaultBrowserOnboarding.goToSettingsPressed.add()
+        // Onboarding
+        case (.action, .press, .dismissedOnboarding, let slideNum, _):
+            GleanMetrics.Onboarding.finish.record(extra: [.slideNum : slideNum])
```

Comment:
Can you follow the same format of doing `.add` instead of `record`?

Reply:
In this case, we need to .record this as an event rather than a counter because it is encoding some extra information, in this case, the page of onboarding that the user made it to before pressing the "Start Browsing" button.

Reply:
Oh got it!